apps/dashboard/config/routes.rb

```diff
@@ -49,6 +49,13 @@
     end
   end
 
+  # Jobs
+  if Rails.root.join('apps', 'activejobs', 'manifest.yml').file?
```

Comment:
In the files app PR, I have this:

```
if ! Rails.env.production? || File.file?('/var/www/ood/apps/sys/files/manifest.yml')
```

here we should probably do the same:

```
if ! Rails.env.production? || File.file?('/var/www/ood/apps/sys/activejobs/manifest.yml')
```

Reply:
And then look back at the nginx config to see if an env var is set that specifies `/var/www/ood/apps/sys` but that can be done via a separate issue+PR.

Reply:
:+1: